File: app/src/main/java/tr/name/fatihdogan/books/api/response/UserInfo.java
package tr.name.fatihdogan.books.api.response;

import android.support.annotation.Keep;

import java.util.Date;

@Keep
@SuppressWarnings({"WeakerAccess", "unused"})
public class UserInfo {

    //TODO Documention mentions review but cannot find structure Review review;
    public Copy copy;
    public ReadingPosition readingPosition;
    public Boolean isPurchased;
    public Boolean isInMyBooks;
    public Boolean isPreordered;
    public Date updated;
    public Date acquiredTime;
    public Integer entitlementType;

}

File: app/src/main/java/com/fthdgn/books/api/response/UserInfo.kt
@file:Suppress("unused")

package com.fthdgn.books.api.response

import android.support.annotation.Keep
import java.util.*

@Keep
class UserInfo {

    //TODO Documention mentions review but cannot find structure Review review;
    var copy: Copy? = null
    var readingPosition: ReadingPosition? = null
    var isPurchased: Boolean? = null
    var isInMyBooks: Boolean? = null
    var isPreordered: Boolean? = null
    var updated: Date? = null
    var acquiredTime: Date? = null
    var entitlementType: Int? = null

}
